DMCompiler/DMCompiler.cs

```diff
@@ -150,6 +157,12 @@ public static class DMCompiler {
         public static void Error(CompilerError error) {
             Console.WriteLine(error);
             ErrorCount++;
+            if (ErrorCount >= 200)
+            {
+                Console.WriteLine("Maximum compilation errors exceeded, aborting.");
+                EndCompilation(false);
+                Environment.Exit(1);
```

Comment:
`Environment.Exit()` isn't ideal as it leads to control never being returned to the code that called `DMCompiler.Compile()`